### DISCUSSION TITLE
Connect to cloudlog with internal IP and domain name

### DISCUSSION THREAD

**Discussion by thegreatcowboy66** [magicbug/Cloudlog#3173] [Q&A]
I setup cloudlog last night and i have yet to figure out how to port foward the web interface outside of my network. I have it on a rpi5 with the ip of 192.168.1.109. I want to forward it outside so that I can access it through my domain name na13cw.com. im gonna use port 8080.

when I tried to access the website through na13cw.com:8080 it redirected me to 192.168.1.109:8080, the problem is i was outside of my network. if i change  /var/www/html/application/config/config.php line 104 "$config['base_url'] = 'http://192.168.1.109:8080';" to "$config['base_url'] = 'na13cw.com:8080';" i can no longer access it inside my network, but i can outside of it. any help?

**Comment by Reset4350** [comment]
Can you back out the change you made to config.php, verify operation on port 80 on your internal LAN. If all is good, set the port forward on your router to forward WAN port 8080 to LAN port 80. See if that helps.
